src/fundus/publishers/de/winfuture.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class winfutureParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//div[@id='news_content']")
+        _summary_selector = XPath("//div[@id='news_content']/div[@class=teaser_text]")
+        _subheadline_selector = XPath("//div[@id='news_content']/h2")
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                doc=self.precomputed.doc,
+                paragraph_selector=self._paragraph_selector,
+                summary_selector=self._summary_selector,
+                subheadline_selector=self._subheadline_selector,
+            )
```

Comment:
Unfortunately, the layout used by this newspaper is so uncommon, that you can't rely on this utility function. The article isn't extracted as paragraphs and the layout isn't in order.